### DISCUSSION TITLE
Diode orientation question

### DISCUSSION THREAD

**Discussion by RyanParsley** [cardonabits/haxo-hw#26] [Q&A]
I'd like to solder my own surface mount components. Can I assume all the diodes are oriented the same way with respect to the silkscreen labels? 

I'm fairly novice to electronics so if there's an easy way to deduce the right polarity, I don't know it.

**Comment by jcard0na** [comment]
Hi Ryan,
The silkscreen indicates the diode orientation, yes, but if you are hand soldering, and if your goal is to learn more about electronics, I recommend you compare with the schematic to be triple sure that you are soldering them in the right orientation.  Flipped diodes, specially when there are so many of them, are not fun!
